src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -353,9 +353,9 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
       i_met_lit = i_litr_min
       floating_cn_ratio_decomp_pools(i_met_lit) = .true.
       decomp_cascade_con%decomp_pool_name_restart(i_met_lit) = 'litr1'
-      decomp_cascade_con%decomp_pool_name_history(i_met_lit) = 'LITR1'
-      decomp_cascade_con%decomp_pool_name_long(i_met_lit) = 'metabolic'
-      decomp_cascade_con%decomp_pool_name_short(i_met_lit) = 'MET'
+      decomp_cascade_con%decomp_pool_name_history(i_met_lit) = 'MET_LIT'
+      decomp_cascade_con%decomp_pool_name_long(i_met_lit) = 'metabolic litter'
+      decomp_cascade_con%decomp_pool_name_short(i_met_lit) = 'L1'
```

Comment:
Here and elsewhere I updated `decomp_pool_name_history` and `decomp_pool_name_long` with descriptive names rather than numbers. However:
- I kept `decomp_pool_name_restart` unchanged because otherwise I lost the bfb answers with baseline.
- I returned `decomp_pool_name_short` to the original non-descriptive names to maintain the option of truly short names. These came in handy for certain history fields that otherwise triggered an error by exceeding some length.

Reply:
This also means having to update the master history list that shows up in the documentation:
`doc/source/users_guide/setting-up-and-running-a-case/master_list_file.rst`
I will wait to confirm that this is what you want me to do @wwieder @ekluzek

Reply:
I realized this because 6 cheyenne tests failed with the error
`LITR1C_TO_SOIL1C in fincl(          22 ) for history tape            1  not found`

Otherwise both test-suites were OK.

Reply:
Yes, this sounds.